### PR TITLE
Update test dependencies

### LINKED ISSUE #34: chore: Update dependencies
Python is getting updated, [related](https://github.com/kytos-ng/kytos/issues/448). Some dependencies from this NApp need to be updated.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #34  

### Summary

Updated test dependencies
Depends in kytos [PR](https://github.com/kytos-ng/kytos/pull/460)

### Local Tests

Test by running with updated kytos

### End-to-End Tests
TBA

### CODE REVIEW

File: db/models.py [hunk 1]
```diff
@@ -114,22 +117,23 @@ def validate_intructions(cls, values):
...
                 raise ValueError(msg)
-        return values
+        return self
 
 
 class PipelineBaseDoc(DocumentBaseModel):
     """Base model for Pipeline documents"""
 
-    status = "disabled"
+    status: str = Field(default="disabled")
```
Comment (viniarck) [inline db/models.py]: wouldn't just `status: str = "disabled"` suffice here?

File: requirements/dev.in [hunk 2]
```diff
@@ -1,3 +1,3 @@
--e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]
+-e git+https://github.com/kytos-ng/python-openflow.git@upgrade/python#egg=python-openflow
```
Comment (viniarck) [inline requirements/dev.in]: reminder to revert the pinned branch

**Review by viniarck — COMMENTED** [review]
Overall looks good, just one minor question though.

File: requirements/dev.in (hunk 2 above)
Comment (Alopalao) [reply requirements/dev.in]: Reverted

File: db/models.py (hunk 1 above)
Comment (Alopalao) [reply db/models.py]: Yes, it did. Changed

**Review by viniarck — APPROVED** [review]
LGTM. Leaving it pre-approved to land soon.

**Pull request merged**